lib/features/manage_account/presentation/menu/manage_account_menu_view.dart

```diff
@@ -16,138 +16,106 @@ class ManageAccountMenuView extends GetWidget<ManageAccountMenuController> {
   Widget build(BuildContext context) {
      return Scaffold(
        backgroundColor: Colors.white,
-       body: SafeArea(right: false, bottom: false,
+       body: SafeArea(
+         right: false,
+         bottom: false,
+         child: Container(
```

Comment:
```suggestion
         child: ColoredBox(
```

Reply:
We have many ways of writing. Just specifying only the color property in a Container will be optimized under the hood to ColoredBox by default now.